packages/rax/src/markupChecksum.js

```diff
@@ -0,0 +1,53 @@
+const MOD = 65521;
+const CHECKSUM_ATTR_NAME = 'data-rax-checksum';
+const TAG_END = /\/?>/;
+
+// adler32 is not cryptographically strong, and is only used to sanity check that
+// markup generated on the server matches the markup generated on the client.
+// This implementation (a modified version of the SheetJS version) has been optimized
+// for our use case, at the expense of conforming to the adler32 specification
+// for non-ascii inputs.
+export function adler32(data) {
```

Comment:
Checksum is not really work, could avoid compute the checksum value, only append a checksum att.